system-variables.md

```diff
@@ -888,6 +871,16 @@ Query OK, 0 rows affected (0.09 sec)
 - Default value: `ON`
 - This variable is used to dynamically control whether the telemetry collection in TiDB is enabled. By setting the value to `OFF`, the telemetry collection is disabled. If the [`enable-telemetry`](/tidb-configuration-file.md#enable-telemetry-new-in-v402) TiDB configuration item is set to `false` on all TiDB instances, the telemetry collection is always disabled and this system variable will not take effect. See [Telemetry](/telemetry.md) for details.
 
+### tidb_enable_top_sql <span class="version-mark">New in v5.4.0</span>
+
+> **Warning:**
+>
+> Currently, Top SQL is an experimental feature. It is not recommended that you use it for production environments.
+
+- Scope: GLOBAL
+- Default value: `OFF`
+- This variable is used to control whether to enable the [Top SQL](/dashboard/top-sql.md) feature.
+
```

Comment:
This was previously at the bottom of the page (non alphabetical order). Chronological order doesn't make sense to users, so the order is fixed.

Reply:
the  default value of tidb_enable_top_sql is "ON" in v6.0.0